docs/docs-content/enterprise-version/system-management/ssl-certificate-management.md

```diff
@@ -59,18 +62,33 @@ You can update your Palette system address and SSL certificates by using the fol
 
 4. Update your Palette domain in the **System Address (UI and API)** field.
 
-5. Copy and paste the certificate into the **Certificate** field.
+5. Convert the certificate, private key, and Certificate Authority (CA) to base64-encoded strings. In the following
```

Comment:
I think it could be helpful to say that CA is often also referred to as root certificate or CA certificate. 

I know these terms are often used interchangeably, but this tripped me up when documenting trusted boot. Technically the CA is not a file, but an entity, and they are trusted to issue a top-level certificate called the root certificate or CA certificate. I think this could help people who are maybe not as familiar with the terminology, because they could have a file they call root certificate and don't know what a CA is.

Reply:
Updated and clarified that it's the certificate authority chain.